setup.cfg

```diff
@@ -27,7 +27,7 @@ exclude_lines =
     pragma: no cover
     raise NotImplementedError
     return NotImplemented
-fail_under = 95
+fail_under = 59
```

Comment:
@jsangmeister Is this value not checked in the ci? Locally it gave Ralf errors, I guess it is not in sync with the GH actions. Either a `--fail-under` is missing [here](https://github.com/OpenSlides/openslides-backend/blob/master/dev/run-tests.sh#L9), or I wonder why the marked line is not respected in the first case (maybe this file is not in the container?)

Reply:
Yes, it is. See https://github.com/OpenSlides/openslides-backend/pull/323/checks?check_run_id=1418394701, section `Run tests`, line 2393. It is also the correct value, in my local instance (and also in GH actions, as you can see) the coverage is over 98%. Probably a configuring error...